test/system/850-compose.bats

```diff
@@ -6,8 +6,8 @@
 load helpers
 
 @test "podman compose - smoke tests" {
-    fake_compose_bin="$PODMAN_TMPDIR/fake_compose"
-    cat >$fake_compose_bin <<EOF
+  fake_compose_bin="$PODMAN_TMPDIR/fake_compose"
+  cat >$fake_compose_bin <<EOF
```

Comment:
these formatting changes are totally unrelated, please drop them as this bloats the diff for no reason

Reply:
Reverted